packages/palette-docs/content/docs/elements/inputs/Select.mdx

```diff
@@ -27,7 +30,7 @@ of user-benefitting product strategy.
   <Serif size="3">Title</Serif>
   <Serif size="2">Short description</Serif>
   <LargeSelect
-    options={[
+    options={\[
```

Comment:
I think those are stray elements here...

Reply:
Yeah, for some reason the rich text editor mode in netlify-cms adds those. 

Need to file an upstream issue (or possibly update netlify-cms)